resources/views/projects/partials/overview.blade.php

```diff
@@ -4,7 +4,11 @@
 
 <h3>{{ __('Project impact') }}</h3>
 
-<h4>{{ __('Communities this project hopes to engage and how they will be impacted') }}</h4>
+@if ($project->checkStatus('published'))
+    <h4>{{ __('Communities this project hopes to engage and how they will be impacted') }}</h4>
+@else
+    <h4>{{ __('How the disability and Deaf communities will be impacted by the outcomes of this project') }}</h4>
+@endif
```

Comment:
Not sure about this change?

Reply:
In the Figma design, if the user is the publisher and have edit power, it appears as "How the disability and Deaf communities will be impacted by the outcomes of this project", and if the user is viewing the page as public, the text changes to "Communities this project hopes to engage and how they will be impacted".

Reply:
@cherylhjli This doesn't seem correct— because the preview should show what public viewers would see. Can you review the design?

Reply:
Thanks for that catch @chosww ! The correct copy for both is: "How the disability and Deaf communities will be impacted by the outcomes of this project"